src/components/list/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC, PropsWithChildren } from 'react';
+import { List as ChakraList, ListItem } from '@chakra-ui/react';
+
+const List: FC<PropsWithChildren> = ({ children }) => {
+  const isSingleChild = !Array.isArray(children);
+  const childrenArray = isSingleChild ? [children] : children;
+
+  return (
+    <ChakraList spacing="md">
+      {childrenArray.map((child, index) => {
+        return <ListItem key={`listItemChild-${index}`}>{child}</ListItem>;
```

Comment:
I would prefer to export the ListItem component and not hardcode it in the List component. The reasoning is that I'd like to achieve the structure:
```html
<ul>
  <li>some text</li>
  <li>some text</li>
</ul>
```
without having a `p` or `span` etc.

Reply:
You can achieve that if you pass strings as children
```
  <List>
    {'asdas'}
    {'asda'}
    {'asda'}
  </List>
```